huaweicloud/services/acceptance/dds/resource_huaweicloud_dds_instance_v3_test.go

```diff
@@ -82,100 +116,38 @@ func TestAccDDSV3Instance_basic(t *testing.T) {
 
 func TestAccDDSV3Instance_withEpsId(t *testing.T) {
 	var instance instances.InstanceResponse
-	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	rName := acceptance.RandomAccResourceName()
```

Comment:
ditto

Reply:
don't need, RandomAccResourceName() is fine